CHANGELOG.md

```diff
@@ -37,6 +37,24 @@
 
     With this release, esbuild will now only merge adjacent duplicate rules together if they are known to work in all browsers (specifically, if they are known to work in IE 7 and up). Adjacent duplicate rules will no longer be merged in all other cases including modern pseudo-class selectors such as `:focus`, HTML5 elements such as `video`, and combinators such as `a + b`.
 
+* Minify syntax in the CSS `font`, `font-family`, and `font-weight` properties ([#1756](https://github.com/evanw/esbuild/pull/1756))
+
+    This release includes size reductions for CSS font syntax when minification is enabled:
+
+    ```css
+    /* Original code */
+    div {
+      font: bold 1rem / 1.2 "Segoe UI", sans-serif, "Segoe UI Emoji";
+    }
+
+    /* Output with "--minify" */
+    div{font:700 1rem/1.2 Segoe UI,sans-serif,"Segoe UI Emoji"}
+    ```
```

Comment:
Hi, @sapphi-red 

`font-family: "Segoe UI Emoji"`

What is the reason for not removing the quotes when compressing? As far as I know, it is safe to remove the quotes.

Reply:
Hi.

1. [`emoji` is a `<generic-family>`](https://www.w3.org/TR/css-fonts-4/#emoji-def).
1. if the font name includes `<generic-family>`, it must be quoted.

> Any identifier which could be misinterpreted as a pre-defined keyword in the font-family value definition, or the common-keywords|CSS-wide keyword values, is not allowed.
> Note: this means that if you really have a font whose name is the same as one of the [<generic-family>](https://www.w3.org/TR/css-fonts-4/#generic-family-value) names, or the common-keywords|CSS-wide keyword values, it must be quoted.
> https://www.w3.org/TR/css-fonts-4/#family-name-syntax

This is the reason.

Reply:
Thanks, I forgot that `emoji` is also `<generic-family>`.